pkg/agent/util/net_windows.go

```diff
@@ -460,10 +460,26 @@ func PrepareHNSNetwork(subnetCIDR *net.IPNet, nodeIPNet *net.IPNet, uplinkAdapte
 			hnsNetworkDelete(hnsNet)
 		}
 	}()
-
-	adapter, ipFound, err := adapterIPExists(nodeIPNet.IP, uplinkAdapter.HardwareAddr, ContainerVNICPrefix)
+	var adapter *net.Interface
+	var ipFound bool
+	err = wait.PollImmediate(time.Second, 3*time.Second, func() (bool, error) {
```

Comment:
Why use 3s as the timeout? What is the expected cost for DHCP request/reply in your local test? 
Can we reduce the poll interval with 500ms even 200ms?

Reply:
Obtaining a valid IP requires more than 1300 milliseconds on my testbed.